lib/bolt/module.rb

```diff
@@ -42,6 +42,12 @@ def plugin_data_file
     end
 
     def plugin?
+      if File.exist?(File.join(path, 'bolt-plugin.json'))
+        msg = "Found bolt-plugin.json in module #{name}. Bolt looks for " \
```

Comment:
```suggestion
        msg = "Found bolt-plugin.json in module #{name} at #{path}. Bolt looks for " \
```

Just to make it a little easier for users to find that module.

Reply:
Sweet updated.